src/main/java/MainApplication.java

```diff
@@ -0,0 +1,15 @@
+import twitter4j.*;
+
+public class MainApplication {
+    public static void main(String[] args) throws TwitterException {
+        createTweet("My First tweet");
```

Comment:
Get user input for the tweet.

Reply:
both paging value and new tweet will be given by the user in next commit